strax/dtypes.py

```diff
@@ -103,6 +103,47 @@ def record_dtype(samples_per_record=DEFAULT_RECORD_LENGTH):
 ]
 
 
+def hitlet_dtype(n_widths=11, n_sample=0):
+    """
+    Hitlet dtype same as peaklet or peak dtype but for hit-kind of
+    objects.
+
+    Note:
+        If n_samples is set to zero returns an array without "data"-field.
+    """
+    dtype = interval_dtype + [
+        (('Original length of the hit interval in samples',
+          'hit_length'), np.int32),
+        (('Total hit area in pe',
+          'area'), np.float32),
+        (('Maximum of the PMT pulse in pe/sample',
+          'amplitude'), np.float32),
+        (('Position of the Amplitude in ns (minus "time")',
+          'time_amplitude'), np.float32),
+        (('Hit entropy',
+          'entropy'), np.float32),
+        (('Peak widths in range of central area fraction in ns',
+          'width'), np.float32, n_widths),
+        (('Peak widths: time between nth and 5th area decile in ns',
+          'area_decile_from_midpoint'), np.float32, n_widths),
+        (('FWHM of the PMT pulse in ns',
+          'fwhm'), np.float32),
+        (('Left edge of the FWHM in ns (minus "time")',
+          'left'), np.float32),
+        (('FWTM of the PMT pulse in ns', 'fwtm'),
+         np.float32),
+        (('Left edge of the FWTM in ns (minus "time")',
+          'low_left'), np.float32),
+        (('Fragment index in which hit was found (hitlets can be in more than 1 fragment)',
+          'record_i'), np.int32)]
+
+    if n_sample:
```

Comment:
Do you foresee any problems if we change this as it won't be reflected in the lineage hash while it might render your data incompatible with what you expect to load as the dtype doesn't match.

Reply:
Good catch have not thought about this. I will change it.